core/Dockerfile

```diff
@@ -46,5 +46,10 @@ COPY --from=downloadBinaries \
 # Copy frontend built on frontendBuild to this stage
 COPY --from=frontendBuilder /home/pi/frontend/dist /home/pi/frontend
 
+# Asserts
+## Make sure that .config does not exist, this needs to be populated at runtime,
+## any file that exists under this folder will not be accessible after the docker bind.
+RUN [ ! -d "/root/.config" ]
+
```

Comment:
To better understand, what will be the final behavior with this change?
If we have something under `/root/.config`, what will happen? And if it is not there?

Reply:
As the comment says, is an Assert. If the folder exists the build will be canceled.

Reply:
Just the bind will be canceled, or all the building?

Reply:
Build, is a docker step.

Reply:
So, after the folder is created the first time, it would not work anymore? I think I'm missing something.

Reply:
As I said, it's an assert. The build will fail if the folder exist. We do not want the folder to exist, because if it does exist we are doing something wrong since the path will have a bind in the host computer and everything inside will not be available for it.

Reply:
Oh, it checks if the folder exists *inside* the docker image. Of couse. Got it upside down.